modules/master-node-sizing.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id="master-node-sizing_{context}"]
+=  Master node sizing
+
+Control plane density focused tests were run at different node scale to determine the master node resource requirements depending on the number of nodes in the cluster. The recommended sizes are
+
+[options="header",cols="3*"]
+|===
+| Node count |CPU cores |Memory (GB)
+
+| 25
+| 4
+| 16
+
+| 100
+| 8
+| 32
+
+| 250
+| 16
+| 64
+
+|===
+
+[IMPORTANT]
+====
+In {product-title} {product-version}, the master node size cannot be modified on a running cluster. It is highly recommended to estimate the node count and use the suggested sizes.
```

Comment:
Maybe 
> Because you cannot modify the master node size in a running {product-title} {product-version} cluster, you must estimate your total node count and use the suggested master size during installation.